.github/CODEOWNERS

```diff
@@ -2,21 +2,21 @@
 ##### Global Protection Rule ######
 ###################################
 # NOTE: This rule is overriden by the more specific rules below. This is the catch-all rule for all files not covered by the more specific rules below
-*                                               @hashgraph/hedera-smart-contracts-product
+*                                               @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts
 
 #########################
 ##### Hedera Relay ######
 #########################
-/packages/relay/                                @AlfredoG87 @ebadiere @lukelee-sl @hashgraph/hedera-smart-contracts-product
-/packages/server/                               @AlfredoG87 @ebadiere @Nana-EC @hashgraph/hedera-smart-contracts-product
-/packages/ws-server/                            @AlfredoG87 @ebadiere @georgi-l95 @Ivo-Yankov @hashgraph/hedera-smart-contracts-product
+/packages/relay/                                @AlfredoG87 @lukelee-sl @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts
+/packages/server/                               @AlfredoG87 @Nana-EC @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/hedera-smart-contracts
```

Comment:
Alfredo and luke are both in hedera-smart-contracts, so remove the individual user callouts.